config.yaml.example

```diff
@@ -12,17 +12,17 @@ verbosity               : DEBUG  # Verbosity of log messages in the console.
                                  # Full log will always be available in a file.
 
 # RNG Seed (requires setting vars.set_seed: True)
-rng_seed_num            : 160    # If starting a new run and using set_seed, use this RNG seed
+rng_seed_num            : 31     # If starting a new run and using set_seed, use this RNG seed
 use_favored_seed        : False  # If starting a new run and not using set_seed, use a curated array of seeds
 
 # Run configuration variables in vars.py
 vars:
   # Important Values
   artificial_pauses     : False  # Set depending on hardware. True == less powerful hardware.
   csr_value             : True   # Set automatically on new game. For testing (loading a save file) set for your environment.
-  nemesis_value         : False  # Set based on if you're doing any% (False) or Nemesis% (True)
+  nemesis_value         : True   # Set based on if you're doing any% (False) or Nemesis% (True)
```

Comment:
Most of the changes in this file likely shouldn't happen. We really only need the change that is providing a sane default for the new config value.